src/content/app-platform/create-catalog/index.md

```diff
@@ -0,0 +1,128 @@
+Kubernetes cluster. At Giant Swarm we use GitHub Pages and this is what we will
+cover in this guide.
+
+## Creating an app catalog using GitHub Pages
+
+First you should choose a name for your catalog. We recommend using the suffix
+`-catalog` to make it clear this repository hosts an app catalog.
+
+Create the git repository in GitHub and enable [GitHub Pages](https://docs.github.com/en/pages/quickstart)
+for the `main` branch.
+
+Initialize the empty helm index.yaml with `helm repo index .` and commit it to
+the repository.
+
+## Pushing an app to an app catalog
+
+TODO
+
+I think we should recommend using ABS (app-build-suite) and this also means our
+metadata extensions will work.
+
+But I have some questions / concerns.
+
+- Should we link to the ABS [tutorial](https://github.com/giantswarm/app-build-suite/blob/master/docs/tutorial.md)?
+- Or should we create another docs page and link them?
+- ABS doesn't commit the files to the repo (which is intentional) but I think we need a solution for that.
+- We could suggest architect-orb but that feels weird since its mainly internal and customer may not be using
+Circle CI.
+- Should we create a simple GitHub Action for this?
+
+## Create catalog CR
+
+Once you've created your app catalog you need to create a catalog CR in the
+management cluster to register it with app platform. This can be done using
+[kubectl gs template catalog]({{< relref "/ui-api/kubectl-gs/template-catalog" >}}).
+
+You should create the catalog CR in the organization namespace where the apps
+will be used.
+
+```nohighlight
+kubectl gs template catalog \
+  --name example-catalog \
+  --namespace org-example \
+  --description "An example Catalog" \
+  --url https://example.github.io/my-app-catalog/ \
+  --logo https://example.com/logos/example-logo.png
+```
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: Catalog
+metadata:
+  name: example-catalog
+  namespace: org-example
+spec:
+  description: An example Catalog
+  logoURL: https://example.com/logos/example-logo.png
+  storage:
+    URL: https://example.github.io/my-app-catalog/
+    type: helm
+  title: example-catalog
+```
+
+## Referencing an app catalog in an app CR
+
+When the catalog CR is not in the `default` namespace you need to set the catalog
+namespace to the organization namespace where it is stored. This can be done using
+[kubectl gs template app]({{< relref "/ui-api/kubectl-gs/template-app" >}}).
+
+```nohighlight
+kubectl gs template app \
+  --catalog example-catalog \
+  --catalog-namespace org-example \
+  --name example-app \
+  --namespace default \
+  --cluster 2hr7z  \
+  --version 0.1.0
+```
+
+```yaml
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  name: example-app
+  namespace: 2hr7z
+spec:
+  catalog: example-catalog
+  catalogNamespace: org-example
+  kubeConfig:
+    inCluster: false
+  name: example-app
+  namespace: default
+  version: 0.1.0
+```
```

Comment:
Should we also include info about how `ace`s are generate?

Reply:
Good point. I'll add.

Reply:
Done. Last section explains this and shows listing ACE CRs for catalog with kubectl.